biogeochem/EDLoggingMortalityMod.F90

```diff
@@ -418,6 +433,237 @@ end subroutine get_harvest_rate_area
 
    ! ============================================================================
 
+   subroutine get_harvestable_carbon (csite, site_area, hlm_harvest_catnames, harvestable_forest_c )
+
+     !USES:
+     use SFParamsMod,  only : SF_val_cwd_frac
+     use EDTypesMod,   only : AREA_INV
+
+
+     ! -------------------------------------------------------------------------------------------
+     !
+     !  DESCRIPTION:
+     !  get the total carbon availale for harvest for three different harvest categories:
+     !  primary forest, secondary mature forest and secondary young forest
+     !  under two different scenarios:
+     !  harvestable carbon: aggregate all cohorts matching the dbhmin harvest criteria
+     !  available carbon: aggregate all cohorts
```

Comment:
Is this 'available' vs 'harvestable' carbon logic still used? I think it isn't so suggest changing the description.

```suggestion
     !  according to the the dbhmin harvest criteria
```

Reply:
Yeah. I forgot changing some of the comment here. Thank you for double checking.